packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -2319,7 +2319,91 @@ describe('DirectStripeRoutes', () => {
     });
   });
 
-  describe('updatePayment', () => {});
+  describe('updatePayment', () => {
+    let request;
+    const paymentToken = 'tok_visa';
+
+    beforeEach(() => {
+      request = request = { ...VALID_REQUEST, payload: { paymentToken } };
+
+      directStripeRoutesInstance.stripeHelper.updateCustomerPaymentMethod.returns();
+    });
+
+    describe('when the customer exists', () => {
+      it('updates the payment method', async () => {
+        const customer = Object.create(customerFixture);
```

Comment:
I think there's no need to new up an object here.  I don't see a reason why passing `customerFixture` on the next line wouldn't work.

Reply:
I can update. Thanks :)